event/serializers.py

```diff
@@ -2,7 +2,7 @@
 from rest_framework import serializers
```

Comment:
@ZenabKhan need to also change the event seriliazer?

Reply:
@mashhoodr you mean to add EventTag in serializer?

Reply:
yes so it returns the tag object not just the id